src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -211,7 +211,8 @@ public void getAllStudents(ResourceRequest request, ResourceResponse response) {
       logger.info("Getting all student list");
       studentList = studentService.getStudentList();
     } catch (Exception e) {
-      logger.error("Error while getting all students", e);
+      //logger.error("Error while getting all students", e);
+	    e.printStackTrace();
     }
     Map<String, Object> map = new HashMap<String, Object>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://13.59.219.140:9000/coding_rules#rule_key=squid%3AS2293)